docs/userguide/transforms/general/index.md

```diff
@@ -51,8 +54,13 @@ transliteration is not translation. Rather, script transliteration it is the
 conversion of letters from one script to another without translating the
 underlying words. The following shows a sample of script transliteration:
 
-Source Transliteration キャンパス kyanpasu Αλφαβητικός Κατάλογος Alphabētikós
-Katálogos биологическом biologichyeskom * Some of the characters may not be
+| Source | Transliteration |
```

Comment:
optional: as far as it makes sense, i prefer tables to be readable in .md too, with aligned columns

Reply:
I aligned the tables in places where it made sense, but in this doc in particular, most of the time, it's futile to make these aligned because the Japanese characters don't work with the monospace font